webpack/react_app/extend/fills.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { addGlobalFill } from 'foremanReact/components/common/Fill/GlobalFill';
+import RecentJobsCard from '../components/RecentJobsCard';
+
+export default addGlobalFill(
+  'details-cards',
+  '[REX]host-details-latest-jobs',
```

Comment:
Can we loose the `[]` it seems weird :thinking: 
```suggestion
  'REX-host-details-latest-jobs',
```